doc/api/repl.md

```diff
@@ -504,8 +508,8 @@ changes:
     the input was incomplete and prompt for additional lines.
   * `useColors` {boolean} If `true`, specifies that the default `writer`
     function should include ANSI color styling to REPL output. If a custom
-    `writer` function is provided then this has no effect. **Default:** the
-     REPL instances `terminal` value.
+    `writer` function is provided then this has no effect. **Default:** the REPL
+    instances `terminal` value and, if available, checks `output.hasColors()`.
```

Comment:
Two things, one small and one a bit more signficiant.

Small thing: While in here, can we fix the typo?

```suggestion
    instance's `terminal` value and, if available, checks `output.hasColors()`.
```

Bigger thing: This sentence is confusing because I don't understand how "checks `output.hasColors()`" affects the default value.  Is there a way to move the "if available, checks `output.hasColors()`" stuff somewhere else, or to clarify if/how that affects the Default value?

Reply:
Like, OK, it checks `output.hasColors()`....and what happens if the check succeeds? Fails? Does that override the instance's `terminal` value? Or is it only used if `terminal` isn't set or is inconclusive? (I mean, I can go read the code, but I would want to know from just reading the docs.)

Reply:
It checks `repl.useColors = options.useColors === undefined ? repl.terminal && (!output.hasColors || output.hasColors()) : options.useColors`.

Finding brief words for it seems quite challenging.